### DISCUSSION TITLE
Turn off the PC => So long time

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1131] [General]
Hello all,

Since some weeks ago, turning off my computer is a nightmare.
The process takes more than 3 minutes.. I don't know what happens, I have only GO installed in it.

I completely reinstalled the system Linux Mint Cinnamon Edition 20.3, reinstalled GO and nothing changes.
Startup is quick, the organ is loaded near 35 seconds, but turning off takes each time 2-3 minutes.

In Linux Mint forum they didn't give me any answer. They advise me to update the bios .. I made it but had many troubles .. it is after that I completely reinstalled the system..

Could you help me ? Maybe there is a long process during the turn off .. I don't know ..
Many thanks

**Comment by oleg68** [comment]
Could you press the up narrow key when your computer is shutting down, make a photo of the screen and post here

**Reply by Jean-Gabriel1982** [reply]
Nothing happens. I just have the Linux Mint logo appearing on the screen ..

**Comment by midimusic** [comment]
Is your computer connected to the internet? If so disconnect it and see if there is any difference.
csw900

**Reply by Jean-Gabriel1982** [reply]
No it is not connected to internet .. 

Even if I connect it, the problem is the same.

**Comment by thbeckpr** [comment]
I use Mint 20.3 with GO and I don't have that problem at all, so I think it might be something specific to your system.

**Reply by Jean-Gabriel1982** [reply]
Yes I think so but the question is what is the problem ... ;-)

**Comment by midimusic** [comment]
If you start your computer and DO NOT run GO or anything else and then turn it off, does it turn off normally? If so the problem is probably in GO otherwise it is not.

You could also try running some other program INSTEAD of GO to see whether the problem persists with other programs.

You could also try testing your memory - bad memory can cause no end of weird problems. Including unwanted and persistent software loops which is a likely cause of a slow turn-off..

csw900

**Reply by Jean-Gabriel1982** [reply]
```
You could also try testing your memory - bad memory can cause no end of weird problems. Including unwanted and persistent software loops which is a likely cause of a slow turn-off..

```

How I can do to test the memory ?
Starting and shutting off the computer without GO does not solve the problem.

**Comment by Jean-Gabriel1982** [comment]
I partially resolved the problem ... for now ...
Just changed the kernel from 5.13 to new 5.17 one ..  I tested twice, with GO and without GO and my computer turns off in 2 seconds ... as before :)

Just one thing I have to check, is I have a broken package during  the 5.17 kernel installation.
The system send me an alert each time I launch the update manager ..

**Comment by midimusic** [comment]
There may be a bug in the 5.13 Kernel!!  However it is always good to know that your memory has no faults so I would still test it. One way to test Linux memory ---- 

Install the memtester utility from the software repository, if it is not already installed. Ubuntu and Linux Mint users type "sudo apt-get install memtester" at the command line.
Type the command "sudo memtester 100 5" to test the memory. Replace "100" with the size, in megabytes, of the RAM installed on the computer. Replace "5" with the number of times you want to run the test.

Please note that this tester is unlikely to test the memory occupied by the operating system.

csw900

**Reply by Jean-Gabriel1982** [reply]
Ok i run the command but .. I have 64Go ram ... that is 64.000Mo that's it ?

When I turn memtester with 64000 1 it gives the result 

> Trying mlock... process stopped

But when I put 6400 1 ... 
>  Trying mlock ... locked

And the test begin ...

In the System info I have 62.24GiB Ram, used 6,78 GiB

And for the broken package :
![image](https://user-images.githubusercontent.com/20370038/169378632-242416f4-f3a0-4917-a2c7-75f52974fbe4.jpeg)

**Comment by midimusic** [comment]
I am amazed that your system appears to be using nearly 7GB of memory. You must have some large programs running. I have used memtester on my Fedora system and it uses less than 2GB. 

Memtester can test only memory which can be allocated to it by the system thus for your system with the figures you have given this is 62.24 - 6.78 - a bit more for memtest and a bit for spare. I would try 55G 1 and see if it accepts that. It is very thorough and testing takes a long time.

I cannot help with the broken package but it may have broken because of the fault on your system. It seems to me that you need to test the low memory which the system is using so you may have to find a memory tester which runs before the system loads. Try Google - I only know how to do this on Windows. It may be worth swapping your memory cards around and doing some more memory tests - but remember where they were to start with.

csw900

**Reply by Jean-Gabriel1982** [reply]
I don't know, maybe my system is misconfigured..

Afterwards, the information I gave you does not come from the System Monitor, but from System Information, where all the characteristics of the PC are listed: Operating system with version and version of the kernel, technical characteristics of the motherboard, card graphics, usb etc...

On the PC there is only GO installed. I uninstalled everything Office Suite, Mail, Games, preinstalled...

**Reply by midimusic** [reply]
It's not what is installed that uses memory. Memory is used only by programs that are actually running. And this should be NOTHING other than the system itself which should not normally use more than 2GB. As I said above bad memory can give all kinds of weird symptoms.
csw900

**Reply by Jean-Gabriel1982** [reply]
I check now and I think all is normal no ?

![image](https://user-images.githubusercontent.com/20370038/169590303-c67b70c8-6014-440c-80c1-e0983560e84f.jpeg)

![image](https://user-images.githubusercontent.com/20370038/169590392-297b615f-abe6-48ac-b36d-93d378604261.jpeg)

![image](https://user-images.githubusercontent.com/20370038/169590521-ea9787af-b64e-45bd-84d7-3c4d4a285162.jpeg)

**Reply by midimusic** [reply]
The only thing that looks a bit odd is the 21.9GB cache (which MAY just be a reflection of 60GB of memory). Unless your faults have gone away I still think you need to do some serious memory testing on the basic hardware, before the system starts as indicated above.
csw900
